README.md

```diff
@@ -27,6 +27,38 @@ npm install eslint eslint-plugin-package-json jsonc-eslint-parser --save-dev
 
 ## Usage
 
+### Flat Config
+
+In your ESLint configuration file:
+
+```js
```

Comment:
[Style] Here and the other added block - per the comment in `.eslintrc.cjs`

```suggestion
```ts
```